pkg/cmd/job/job_update.go

```diff
@@ -0,0 +1,57 @@
+package job
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+// newCmdClusterUpdate updates a job.
+func newCmdJobUpdate(ctx api.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "update a job",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := metronomeClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			// Handling input from file or stdin
```

Comment:
Does it make sense to have a helper function for this? The same way that we have `metronomeClient()`, eg. `inputReader()`